modules/mono/csharp_script.cpp

```diff
@@ -516,22 +516,10 @@ static String variant_type_to_managed_name(const String &p_var_type_name) {
 }
 
 String CSharpLanguage::make_function(const String &, const String &p_name, const PackedStringArray &p_args) const {
-	// FIXME
-	// - Due to Godot's API limitation this just appends the function to the end of the file
-	// - Use fully qualified name if there is ambiguity
-	String s = "private void " + p_name + "(";
-	for (int i = 0; i < p_args.size(); i++) {
-		const String &arg = p_args[i];
-
-		if (i > 0) {
-			s += ", ";
-		}
-
-		s += variant_type_to_managed_name(arg.get_slice(":", 1)) + " " + escape_csharp_keyword(arg.get_slice(":", 0));
-	}
-	s += ")\n{\n    // Replace with function body.\n}\n";
-
-	return s;
+	// TODO
+	// Implement function generation. This has been disabled for now because the behaviour was not consistent
```

Comment:
We use en_US spelling:
```suggestion
	// Implement function generation. This has been disabled for now because the behavior was not consistent
```

Reply:
Noted. I'm going to re-write this comment as per @raulsntos' point [above](https://github.com/godotengine/godot/pull/87952#discussion_r1478874974) anyway.